### ISSUE TITLE
Constant disconnect/reconnect cycles after resetting the device

### ISSUE THREAD

**Issue by lileffects** [max22-/ESP32-BLE-MIDI#30]
If I pair my device via BT to my Windows 10 PC everything works fine until I power cycle my ESP32. After power cycling I get stuck in a constant loop of BT connect/disconnect which is only solved by removing a pairing the ESP32 board again. 

Anyone solve this?

**Comment by SwitchMan92** [comment]
yep, same here. If i power reset the esp32 while connected, i get in a constant state of coupling / connected, without ever actually managing to properly connect the device.

platform: windows 10

**Comment by SwitchMan92** [comment]
Alright so what worked for me is reverting to this commit https://github.com/max22-/ESP32-BLE-MIDI/commit/405c493c3832689e5874372ade64bb4007fc5dc1, which is the last commit before migrating to nimble....
and linking the library manually against a local version reverted to the aforementioned commit.
